Libraries/Text/RCTTextAttributes.m

```diff
@@ -61,6 +62,7 @@ - (void)applyTextAttributes:(RCTTextAttributes *)textAttributes
   _lineHeight = !isnan(textAttributes->_lineHeight) ? textAttributes->_lineHeight : _lineHeight;
   _alignment = textAttributes->_alignment != NSTextAlignmentNatural ? textAttributes->_alignment : _alignment; // *
   _baseWritingDirection = textAttributes->_baseWritingDirection != NSWritingDirectionNatural ? textAttributes->_baseWritingDirection : _baseWritingDirection; // *
+  _lineBreakStrategy = textAttributes->_lineBreakStrategy != NSLineBreakStrategyNone ? textAttributes->_lineBreakStrategy : _lineBreakStrategy;
```

Comment:
From @javache:
> textAttributes->_lineBreakStrategy is always valid, why not always apply it?
Also consider if this works correctly when resetting the prop, can you set lineBreakStrategy to standard and later reset it to none?

Reply:
I changed to `_lineBreakStrategy = textAttributes->_lineBreakStrategy != NSLineBreakStrategyNone ?: _lineBreakStrategy;`